pkg/vault/test/foundry/fork/YieldBearingPoolSwapBase.t.sol

```diff
@@ -65,8 +65,9 @@ abstract contract YieldBearingPoolSwapBase is BaseVaultTest {
         _token1Fork = IERC20(ybToken1.asset());
         _token2Fork = IERC20(ybToken2.asset());
 
-        _ybToken1Factor = 10 ** IERC20Metadata(address(ybToken1)).decimals();
-        _ybToken2Factor = 10 ** IERC20Metadata(address(ybToken2)).decimals();
+        // Equivalent to 10^(18+decimalDifference), since decimalDifference = 18 - tokenDecimals.
```

Comment:
Is this comment still right? Maybe explain the 36 here? Something to do with what's going on below; e.g., replacing "_previewDepositWithBuffer" with "_previewWrapExactIn"?

Reply:
The comment is right, because 18 + decimalDiff = 18 + 18 - tokenDecimals = 36 - token decimals

Reply:
Done!